frameworks/FSharp/giraffe/src/App/Custom.fs

```diff
@@ -0,0 +1,81 @@
+﻿module Custom
+
+open App
+open Dapper
+open Giraffe
+open System.Collections.Generic
+open System
+open Models
+open Npgsql
+open FSharp.Control.Tasks
+open Microsoft.AspNetCore.Http
+open System.Text
+
+let application : HttpHandler = 
+    
+    let inline contentLength x = new Nullable<_> ( int64 x )
+
+    let comp = { new IComparer<Fortune> with 
+        member self.Compare(a,b) = String.CompareOrdinal(a.message, b.message)
+    }
+
+    let none: Option<HttpContext> = None
+
+    let json' data : HttpHandler =
+        let bytes = Utf8Json.JsonSerializer.Serialize(data)
+        fun _ ctx -> 
+            ctx.Response.ContentLength <- contentLength bytes.Length
+            ctx.Response.ContentType <- "application/json"
+            ctx.Response.StatusCode <- 200
+            task {
+                do! ctx.Response.Body.WriteAsync(bytes, 0, bytes.Length)
+                return none

```

Comment:
I am guessing you are not using "json" helper function to allow caching of known serialized data bytes? Although "none" can be returned and will work, this is generally wrong usage as the "choice" functions will continue to try match routes etc until it gets "Some ctx", therefore, to prevent testing of further routes etc, best to return "Some ctx"